libMBIN/Source/NMS/BaseTypes/Vector4f.cs

```diff
@@ -3,19 +3,23 @@
 
 namespace libMBIN.NMS
 {
+    // used for XYZW and RGBA type vectors
```

Comment:
I mean, isn't Colour used for RGBA type things?
Also, if it's `w` in the exe, we could change it I guess?
I think this has been like this from the start. Not sure why it is `t` in the first place, but `w` is the standard for the 4th component of a vector.

Reply:
We use Vector4f in our .cs files for when we encounter RGBA (a color model) and XYZW (a 3D point matrix) vectors in the exe.  So our Vector4f  intent can stand for either one of those two types of vector in NMS (I mean it represents two different meaning but we use the same Vector4f.cs file for both because it it convenient).  I just want to comment the code so we know that when we use Vector4f we can mean one or the other type of vector in the structures.

Yes, it is `w` in the exe, I was not sure if it would impact too much all those AMUMSS scripts out there (or other apps I don't know about or use that refer to it thru the EXML representation) that refer to `t` instead.